feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { TranslationService } from '@spartacus/core';
+import {
+  COLOR_CLOSE,
+  COLOR_OPEN,
+  STATUS_CLOSE,
+  STATUS_OPEN,
+} from '@spartacus/customer-ticketing/root';
+import { Card } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-customer-ticketing-details',
+  templateUrl: './customer-ticketing-details.component.html',
+})
+export class CustomerTicketingDetailsComponent implements OnInit {
+  constructor(protected translation: TranslationService) {}
+
+  ticketDetails$ = of({
+    associatedTo: {
+      code: '00000001',
+      modifiedAt: '2022-06-28T00:00:00+0000',
+      type: 'Cart',
+    },
+    availableStatusTransitions: [
+      {
+        id: 'CLOSED',
+        name: 'Closed',
+      },
+    ],
+    createdAt: '2022-06-22T14:37:15+0000',
+    id: '00000001',
+    modifiedAt: '2022-06-22T20:25:02+0000',
+    status: {
+      id: 'OPEN',
+      name: 'Open',
+    },
+    subject: 'test ticket again',
+    ticketCategory: {
+      id: 'COMPLAINT',
+      name: 'Complaint',
+    },
+    ticketEvents: [
+      {
+        author: 'Mark Rivers',
+        createdAt: '2022-06-22T20:25:02+0000',
+        message: 'This is the way',
+      },
+      {
+        author: 'Mark Rivers',
+        createdAt: '2022-06-22T14:37:15+0000',
+        message: 'A message to consider',
+      },
+    ],
+  });
+
+  ngOnInit(): void {}
+
+  prepareCardContent(
```

Comment:
👍

Reply:
to be honest, I'm fine making it generic like this and re-using it for the other card contents, but let me put this on the table.

What if the customer wants to override the specific data that's showing for 'x' column in your 'overview' component because right now it's made that any changes applied to this function will be by the customer will be applied to all. For example, what if they don't want the title for the 2nd column, or the 1st, etc.

But let me know what you think. Again, we can keep this approach.

Reply:
I agree. But, is it possible to add a condition on the text or title based on the title?
if yes, I know it might be little bit difficult. What do you suggest?